docs/message_examples.md

```diff
@@ -102,18 +102,16 @@ Responses:
 - [failure.get](#failuregetscan)
 ## modify/scan
 
-Topic: eulabeia/scan/cmd/director
+Topic: scanner/scan/cmd/director
 ```
 {
-  "message_id" : "ebb40611-c62a-4bd1-a371-d5efce57756e",
+  "message_id" : "e6c53909-6f0b-4312-95e9-5ddcbdf475bb",
   "message_type" : "modify.scan",
-  "group_id" : "ebb40611-c62a-4bd1-a371-d5efce57756e",
-  "created" : 1629896000451593275,
+  "group_id" : "e6c53909-6f0b-4312-95e9-5ddcbdf475bb",
+  "message_created" : 1632480366136386254,
   "id" : "example.scan.id",
-  "values" : {
-    "temporary" : false,
-    "target_id" : "example.target.id"
-  }
+  "temporary" : false,
+  "target_id" : "example.target.id"
```

Comment:
Why are the values not in a values container anymore?

Reply:
oh that's a bug when creating the message_examples